src/main/java/com/stripe/model/LineItem.java

```diff
@@ -73,6 +77,25 @@
   @SerializedName("taxes")
   List<LineItem.Tax> taxes;
 
+  @Getter
+  @Setter
+  @EqualsAndHashCode(callSuper = false)
+  public static class Discount extends StripeObject {
+    /** Discount amount for this line item. */
+    @SerializedName("amount")
+    Long amount;
+
+    /**
+     * A discount represents the actual application of a coupon to a particular customer. It
+     * contains information about when the discount began and when it will end.
+     *
+     * <p>Related guide: <a href="https://stripe.com/docs/billing/subscriptions/discounts">Applying
+     * Discounts to Subscriptions</a>.
+     */
+    @SerializedName("discount")
+    Discount discount;
```

Comment:
Is this supposed to be recursive like this? Like a Discount has a Discount? I feel like i'm missing something.